tests/cases/fourslash/completionsPathsJsonModuleWithAmd.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path="fourslash.ts" />
+
+// @module: amd
+// @resolveJsonModule: true
+
+// @Filename: /project/test.json
+////not read
+
+// @Filename: /project/index.ts
+////import { } from ".//**/";
+
+verify.completionsAt("", [], { isNewIdentifierLocation: true });
```

Comment:
Why doesn't this have a completion?

Reply:
because module resolution strategy isnt "node" and hence json files wont be resolved.